packaging/linux/build_binaries.sh

```diff
@@ -36,13 +36,19 @@ fi
 echo "-tags '$go_tags'"
 
 # Determine the LD flags.
-ldflags=""
+ldflags_client=""
+ldflags_kbfs=""
 if [ "$mode" != "production" ] ; then
   # The non-production build number is everything in the version after the hyphen.
   build_number="$(echo -n "$version" | sed 's/.*-//')"
-  ldflags="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  ldflags_client="-X github.com/keybase/client/go/libkb.PrereleaseBuild=$build_number"
+  kbfs_path="$GOPATH/src/github.com/keybase/kbfs"
```

Comment:
Sorry, what I said before was confusing. I meant to suggest the `$kbfs_repo` variable itself, rather than the path to the symlink that points to it.